test/topology/util.py

```diff
@@ -366,11 +367,18 @@ async def get_coordinator_host(manager: ManagerClient) -> ServerInfo:
     """Get coordinator ServerInfo"""
 
     coordinator_host_id = (await get_coordinator_host_ids(manager))[0]
-    server_id_maps = {await manager.get_host_id(srv.server_id):srv for srv in await manager.running_servers()}
-    coordinator_host = server_id_maps.get(coordinator_host_id, None)
-    assert coordinator_host, \
-        f"Node with host id {coordinator_host_id} was not found in cluster host ids {list(server_id_maps.keys())}"
-    return coordinator_host
+    for s_info in await manager.running_servers():
+        with suppress(Exception):
+            if await manager.get_host_id(s_info.server_id) == coordinator_host_id:
+                return s_info
+    raise AssertionError(f"There is no host with id={coordinator_host_id} in the cluster")
```

Comment:
nit: the previous error provided more potentially useful details -- the list of host IDs present in the cluster.

Reply:
No, it's for suppressing Exception from get_host_id() call:

```python
    async def get_host_id(self, server_id: ServerNum) -> HostID:
        """Get local host id of a server"""
        try:
            host_id = await self.client.get_json(f"/cluster/host-id/{server_id}")
        except Exception as exc:
            raise Exception(f"Failed to get local host id address for server {server_id}") from exc
        return HostID(host_id)
```

Reply:
I don't understand your answer. My complaint is that the message you removed:
```
        f"Node with host id {coordinator_host_id} was not found in cluster host ids {list(server_id_maps.keys())}"
```

was more rich in details compared to the new message
```
f"There is no host with id={coordinator_host_id} in the cluster"
```

Reply:
OK, got it.